blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
I think instead of not adding this, we need to have a way to know that Fastboot is running and remove it if it is. Because, Fastboot takes extra work, and even if it's added by default, some people would remove it, because it's extra work or doesn't fit into the budget of the app.